www/js/lib/uiUtil.js

```diff
@@ -934,7 +946,7 @@ function warnAndOpenExternalLinkInNewTab (event, clickedAnchor) {
         message += ' ' + (translateUI.t('dialog-open-externalurl-newtab') || '(in a new tab)');
     }
     message += '</p><p style="word-break:break-all;">' + clickedAnchor.href + '</p>';
-    systemAlert(message, translateUI.t('dialog-open-externalurl-title') || 'Opening external link', true).then(function (response) {
+    systemAlert(message, translateUI.t('dialog-open-externalurl-title') || 'Opening external link', true, translateUI.t('dialog-cancel') || 'Cancel', translateUI.t('dialog-confirm') || 'Confirm', translateUI.t('dialog-ok') || 'Okay', true).then(function (response) {
```

Comment:
I understand that translation of dialog button should be done inside `systemAlert` . After add `dialog-hide` translation inside `systemAlert`,still `systemAlert` inside `warnAndOpenExternalLinkInNewTab` need an `displayHideOption` argument. `displayHideOption` is the last argument in `systemAlert` because any other position will break the old code. JavaScript by default doesn't support named argument. That's why i would still need to pass the value of `declineConfirmLabel, approveConfirmLabel, closeMessageLabel` manually in this `systemAlert`.

Reply:
You're just using the default values "Cancel" and "Confirm". To use these, you just pass `null.` As it says in the function description, these values are optional and the default will be used if you don't use them or pass `null` for each of them.